helm/robusta/values.yaml

```diff
@@ -634,6 +634,11 @@ kubewatch:
       runAsUser: 1000
     pod: {}
   customServiceAccount: "" # to override the kubewatch service account
+  service:
+    port: 2112
+  serviceMonitor:
+    path: /metrics
+    interval: 15s
```

Comment:
any reason to change the default?
I think we don't need an overriding value here

Reply:
Removed it